api/src/main/java/org/eclipse/microprofile/reactive/messaging/spi/OutgoingConnectorFactory.java

```diff
@@ -72,8 +74,7 @@
  * In the previous configuration, the reactive messaging implementation would need to find the
  * {@link OutgoingConnectorFactory} implementation qualified with the {@link Connector} qualifier with the value
  * {@code acme.kafka} to create the {@code my-channel} <em>subscriber</em>. Note that if
- * the connector cannot be found, the deployment must be failed with a
- * {@link javax.enterprise.inject.spi.DeploymentException}.
+ * the connector cannot be found, the deployment must be failed with a {@link DeploymentException}.
```

Comment:
Don't import the package because of the javadoc. Please use the fully qualified package.

Reply:
It's been a journey to understand how to create external links but got it to work.

Reply:
Done,

I also linked the MP Reactive Streams Operator spec, and MP Config spec.